taxonomy/management/commands/refresh_job_skills.py

```diff
@@ -17,9 +17,11 @@
 
 class Command(BaseCommand):
     """
-        Example usage:
-        Create or refresh the existing data associated with jobs and job skills on the basis
-        of ranking_facet and nested_ranking_facet arguments.
+    Command to refresh the jobs associated with skills.
+
+    Example usage:
+        $ # Create or refresh the existing data associated with jobs and job skills on the basis
+        $ # of ranking_facet and nested_ranking_facet arguments.
         $ ./manage.py refresh_job_skills 'CERTIFICATIONS' 'CERTIFICATIONS_NAME'
```

Comment:
Do we need to something about 'CERTIFICATIONS' 'CERTIFICATIONS_NAME'?

Reply:
@zamanafzal your thoughts on this?